awx/main/access.py

```diff
@@ -980,7 +985,8 @@ def can_add(self, data):
             return check_user_access(self.user, Team, 'change', team_obj, None)
         if data and data.get('organization', None):
             organization_obj = get_object_from_data('organization', Organization, data)
-            return check_user_access(self.user, Organization, 'change', organization_obj, None)
+            return any([check_user_access(self.user, Organization, 'change', organization_obj, None),
+                        self.user in organization_obj.credential_admin_role])
```

Comment:
I really thought that `self.user in organization_obj.credential_admin_role` alone would have been sufficient, but I admit I have not thought about this very hard.